src/Symfony/Component/Serializer/Normalizer/DateTimeNormalizer.php

```diff
@@ -25,6 +25,15 @@ class DateTimeNormalizer implements NormalizerInterface, DenormalizerInterface,
     const FORMAT_KEY = 'datetime_format';
     const TIMEZONE_KEY = 'datetime_timezone';
 
+    /**
+     * In PHP, The $timezone parameter and the current timezone are ignored when the $time parameter either is a UNIX timestamp (e.g. @946684800) or specifies a timezone (e.g. 2010-01-28T15:00:00+02:00).
+     *
+     * The denormalizer assumes that all DateTimeInterface object returned will have the timezone returned by the getTimezone() method.
+     * Default PHP behavior will occur if the getTimezone() method returns null or context[self::PRESERVE_CONTEXT_TIMEZONE] is set to false.
+     * This flag will be ignored in Symfony/Serializer 5.0+
```

Comment:
This flag will be ignored in Symfony 5.0.

Reply:
Done